### ISSUE TITLE
UX: Explore options to advertise the experimental features on the dashboard

### ISSUE THREAD

**Issue by slemeur** [containers/podman-desktop#10225]
### UX Description

When we release new features as experimental, we would like to bring attention to the end-user about those. Obviously, we want people to be trying them out, but we would also like to gather feedback.

The goal of this issue would be to explore the different possibilities to bring some awareness on those experimental features directly on the dashboard.

### Request type

UX analysis/suggestions for improvement

### Primary Contact

@slemeur

**Comment by MariaLeonova** [comment]
Some of the options can be:
 - the most obvious one would be to have a **dismissible banner** - something like this in Podman style:

<img width="928" alt="Image" src="https://github.com/user-attachments/assets/4a787cce-c716-4750-9b9e-59115acebe28" />

- we could also call out specific features with an **interactive tooltip** - it could point to the Settings icon. Very roughly: 

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/220ce83a-8032-40b1-a2d5-6e9be4bd1863" />

- another option would be to create **a number of cards** as a section on the dashboard that would advertise the experimental features, inviting users to try them out
- one more possibility is to have **a modal**, similar to the New Release announcement (possibly an overkill)

@slemeur what would you prefer?

**Comment by deboer-tim** [comment]
My 2 cents: we've talked about having a unified notification system for a while now (want to install kubectl?, Podman needs to be updated, you updated and here are the release notes, your machine is using high CPU, etc, ...) but we've started delivering these as inconsistent one-offs. We should just be extending/improving what we already have, with each type of notification having a different priority/styling/actions, and a consistent way to dismiss (and undismiss) them.

**Comment by MariaLeonova** [comment]
@deboer-tim maybe we could discuss this one at the UX call?

**Comment by slemeur** [comment]
@deboer-tim : I agree if we go down the road of using notifications and "nagging" the users about the experimental features. But is it really what we would like to do here?

A notification could be interesting, when we detect there could be an interest from the user of a specific experimental feature. In this case, it will be a "recommendation" and not be taken as something annoying the user. I believe we want to be cautious with that aspect. 

In this ticket, I think that's exactly what we want to be exploring. From a UX point of view, at what point in time, should we bring attention to some (one or multiple) of the experimental features? 
- What should be the different user flows? 
- Would it make sense to display information on the dashboard about the available experimental features?
- Do we want something more advanced, with a dynamic dashboard component, showcasing them, including feature description, links to demo, to github discussion... ?
Or should we go a different route, have "notifications" and "pop-up tutorials"?

@MariaLeonova Are you aware of solutions that have already been explored and eventually some examples?

**Comment by deboer-tim** [comment]
> [@deboer-tim](https://github.com/deboer-tim) : I agree if we go down the road of using notifications and "nagging" the users about the experimental features. But is it really what we would like to do here?

IMHO the interesting part (as per the rest of your comment) is when do we inform the user, what triggers we use, how we showcase them, etc. I only want to caution about having multiple notification systems, different UI to close or hide or reopen things on the dashboard vs other things we put there, etc. i.e. extend and reuse vs entirely new/independent system.

**Comment by MariaLeonova** [comment]
@slemeur 

> @MariaLeonova Are you aware of solutions that have already been explored and eventually some examples?

Will these be inside Podman Desktop?

**Comment by MariaLeonova** [comment]
I was thinking about this issue and what came to mind as an example of unobtrusive and helpful tips is the following: on Instagram when someone logs in for the first time from a new device or in a new account, it will give you tips on how to use certain features. For example, navigate Stories, create Reels, etc. It shows them only once and usually they are pretty helpful in the moment.
So for this particular case it might be great to do 2 things:
1. show a banner on the dashboard, since the experimental dashboard is new and needs to be advertised
2. show the user a tooltip once (!), saying `Try our new experimental features here!` and pointing to settings and potentially it could include a button.

I think it might be good for them to start with the Experimental features as a concept and from there they can explore individual ones.


Later on, we could also explore options of advertising individual features, too, but it would be good to start broadly to make people aware of where they can access the dashboard and what they can do there.

**Comment by MariaLeonova** [comment]
We can also have an experimental section on the dashboard, next to the learning center, for example, with a carousel.

**Comment by MariaLeonova** [comment]
To illustrate my point above: 

![Image](https://github.com/user-attachments/assets/f5e52d0e-a184-421e-8c1b-e3b5e79c3edb)

I put a banner on top as a suggestion to draw attention to Experimental features as a whole new section, as I believe it is not yet widely known.

If we were to advertise individual features, they could become a part of the release banner and be highlighted there. 

![Image](https://github.com/user-attachments/assets/3915245b-14c7-4339-adaa-3e228d5368f6)

**Comment by MariaLeonova** [comment]
We discussed this issue at the UX meeting today, and agreed to start with advertising  experimental features as part of the Release Announcement. There should be a link to take users to the experimental dashboard, where they can turn it on + it should be very clear  that this feature is experimental. 

I will work on some suggestions.

**Comment by MariaLeonova** [comment]
Here's an example with the Release announcement: the link next to the feature can take users to the dashboard to turn it on:
![Image](https://github.com/user-attachments/assets/6c7fbe97-bae5-4ba0-9303-efc4c9e1678a)

**Comment by slemeur** [comment]
Thanks @MariaLeonova. 
Starting with highlighting the experimental features as part of the release note section is a great first step.

I believe, we should go one step further though and the options you described are making sense. I like the idea of a banner at the top, as it allows to bring some attention to the exp features, but let the user removes that banner too.

The section in the dashboard to draw attention to the experimental features is also interesting, but I feel it takes too much space and might get annoying for some users.

I think we can close the ticket and follow-up with implementation tickets (cc @Firewall )

**Comment by Firewall** [comment]
Yep. It looks great!

I updated the ticket you created (https://github.com/podman-desktop/podman-desktop/issues/11849) @slemeur and added a second one for in the release notes (https://github.com/podman-desktop/podman-desktop/issues/11851) so I think this ticket can be closed 👍

**Comment by MariaLeonova** [comment]
Closing - please create new issues if any other designs are needed.
